### PR TITLE
update policyViolationContextLimit description

### PR DESCRIPTION
**Pull request by ChunxiAlexLuo**
Signed-off-by: Chunxi Luo <chuluo@redhat.com>

### CODE REVIEW

**Review by mprahl — APPROVED** [review]

File: pkg/templates/crds/grc/policy.open-cluster-management.io_policyautomations.yaml [hunk 1]
```diff
@@ -56,8 +56,10 @@ spec:
                     minLength: 1
                     type: string
                   policyViolationContextLimit:
-                    description: The maximum number of violation contexts will be
-                      provided to the Ansible Tower as extra variable.
+                    description: The maximum number of violating cluster contexts
+                      that will be provided to the Ansible job as extra variables.
+                      When policyViolationContextLimit is set to 0, it means no limit.
+                      The default value is 1000.
                     minimum: 0
```
Comment (dhaiducek) [inline pkg/templates/crds/grc/policy.open-cluster-management.io_policyautomations.yaml:63]: This line isn't in the propagator CRD. Should it also be removed?
```suggestion
```

**Review by dhaiducek — APPROVED** [review]

**Comment by dhaiducek** [comment]
@JakobGray Thanks for adding the OWNERS file! We were able to approve this PR. 🎉

**Pull request merged**